assets/js/play.js

```diff
@@ -116,3 +178,4 @@ function setMatchInfo(currentState,
 
   $('#data-online-player-sample img').tooltip();
 }
+*/
```

Comment:
The changes to this file should be added as a new file, eg. `play-fallback.js` for now, as that's easier from a maintenance standpoint.